src/rules/prefer-native-locators.ts

```diff
@@ -0,0 +1,139 @@
+import { getStringValue, isPageMethod } from '../utils/ast'
+import { createRule } from '../utils/createRule'
+
+export default createRule({
+  create(context) {
+    return {
+      CallExpression(node) {
+        const { testIdAttribute } = {
+          testIdAttribute: 'data-testid',
+          ...((context.options?.[0] as Record<string, unknown>) ?? {}),
+        }
+
+        if (node.callee.type !== 'MemberExpression') return
+        const method = getStringValue(node.callee.property)
+        const query = getStringValue(node.arguments[0])
```

Comment:
Should add test + make sure the code works with no args

Reply:
Added in https://github.com/playwright-community/eslint-plugin-playwright/pull/308/commits/4c7ecaf10fd06cd92d3d0d25bb29c179648c7fc4